Drivers/Src/BatteryBalancing.c

```diff
@@ -3,9 +3,12 @@
 #include "Voltage.h"
 #include "BatteryBalancing.h"
 #include "config.h"
-#include "Current.h"
+#include "os.h"
+#include "Tasks.h"
 
-#define CHARGING_TOLERANCE 0
+#define BALANCING_TOLERANCE 50
```

Comment:
Should this be in config.h? Also, if we are balancing between 20-50 mv, should it be around 20 or 30 instead?

Reply:
I agree taht this would be better in config.h. Also, if we want our battery balancing to behave as we described at CDR, we should have two of these (one for starting balancing at >50mV and one for stopping balancing at <20mV).

Reply:
fixed in #274